packages/client/wallets/aa/src/utils/signer.ts

```diff
@@ -0,0 +1,64 @@
+import { CrossmintService } from "@/api";
+import {
+    Blockchain,
+    FireblocksNCWallet,
+    getBlockExplorerByBlockchain,
+    getChainIdByBlockchain,
+    getDisplayNameByBlockchain,
+    getTickerByBlockchain,
+    getTickerNameByBlockchain,
+    getUrlProviderByBlockchain,
+} from "@/blockchain";
+import { FireblocksNCWSigner, UserIdentifier, WalletConfig, Web3AuthSigner } from "@/types";
+import { parseToken } from "@/utils";
+import type { SmartAccountSigner } from "@alchemy/aa-core";
+import { CHAIN_NAMESPACES } from "@web3auth/base";
+import { EthereumPrivateKeyProvider } from "@web3auth/ethereum-provider";
+import { Web3Auth } from "@web3auth/single-factor-auth";
+import { convertEthersSignerToAccountSigner, getRPCProviderOwner } from "@zerodev/sdk";
+import { Signer } from "ethers";
+
+export async function createOwnerSigner(
+    user: UserIdentifier,
+    chain: Blockchain,
+    walletConfig: WalletConfig,
+    crossmintService: CrossmintService
+): Promise<SmartAccountSigner> {
+    if ((walletConfig.signer as FireblocksNCWSigner)?.type === "FIREBLOCKS_NCW") {
+        const passphrase = (walletConfig.signer as FireblocksNCWSigner).passphrase;
+        const fireblocks = await FireblocksNCWallet(user.email, crossmintService, chain, passphrase);
+        return fireblocks.owner;
+    } else if ((walletConfig.signer as Web3AuthSigner)?.type === "WEB3_AUTH") {
+        const signer = walletConfig.signer as Web3AuthSigner;
+        const chainId = getChainIdByBlockchain(chain);
+        const chainConfig = {
+            chainNamespace: CHAIN_NAMESPACES.EIP155,
+            chainId: "0x" + chainId!.toString(16),
+            rpcTarget: getUrlProviderByBlockchain(chain),
+            displayName: getDisplayNameByBlockchain(chain),
+            blockExplorer: getBlockExplorerByBlockchain(chain),
+            ticker: getTickerByBlockchain(chain),
+            tickerName: getTickerNameByBlockchain(chain),
+        };
+
+        const web3auth = new Web3Auth({
+            clientId: signer.clientId, // Get your Client ID from Web3Auth Dashboard
+            web3AuthNetwork: "testnet", // ["cyan", "testnet"]
```

Comment:
planning to do something similiar as done here to define the network? https://github.com/Crossmint/crossmint-sdk/pull/296/files#diff-44a8de940941124bdd920a8721c56d3a0a9262939880fe56f3a457a96c301562R91

Reply:
I was waiting for the other PR merged, I'm doing it right now

Reply:
@maxisch - Done!